README.md

```diff
@@ -370,22 +397,22 @@ gcloud projects add-iam-policy-binding $PROJECT_ID \
 gcloud projects remove-iam-policy-binding $PROJECT_ID \
   --member=serviceAccount:$GCS_SERVICE_ACCOUNT \
   --role roles/pubsub.publisher
-kubectl delete secrets gcssource-key
+kubectl delete secrets gcs-source-key
 kubectl delete services.serving gcs-message-dumper
 ```
 
 # **REST OF THIS DOCUMENT NEEDS UPDATING**
 
-
 ## More complex examples
-* [Multiple functions working together](MULTIPLE_FUNCTIONS.md)
 
+- [Multiple functions working together](MULTIPLE_FUNCTIONS.md)
```

Comment:
Just remove the rest here or better yet, add a quick sample sinking these into a broker :)

Reply:
Deleted and added broker.